### PR TITLE
Add validate when deploy contract

### PR DESCRIPTION
**Pull request by tlatkdgus1**
**What does this PR do?**
Check bytecode when deploy contract

**Why are these changes required?**
can not validate bytecode when deploy contract, so Add Exception about empty bytecode case

**Issue**
#2848

### CODE REVIEW

File: framework/src/main/java/org/tron/core/Wallet.java [hunk 1]
```diff
@@ -350,9 +350,14 @@ public TransactionCapsule createTransactionCapsule(com.google.protobuf.Message m
       CreateSmartContract contract = ContractCapsule
           .getSmartContractFromTransaction(trx.getInstance());
       long percent = contract.getNewContract().getConsumeUserResourcePercent();
+      byte[] data = contract.getNewContract().getBytecode().toByteArray();
+
       if (percent < 0 || percent > 100) {
         throw new ContractValidateException("percent must be >= 0 and <= 100");
       }
+      if (ArrayUtils.isEmpty(data)) {
+        throw new ContractValidateException("bytecode must have");
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/Wallet.java]: `s/bytecode must have/bytecode must not be empty/`

Comment (tlatkdgus1) [reply framework/src/main/java/org/tron/core/Wallet.java]: change this one

**Comment by tlatkdgus1** [comment]
> Hello,The root cause of the problem is that TVM itself can accept and execute empty bytecodes and return a normal execution result. Whether this feature continues to be retained requires further evaluation. If the creation of empty bytecode contracts is prohibited, changes need to be made in TVM.
> Simple verification in Wallet is not enough,this need Hard fork in a future release.

Yes I think so too, it is nothing but a temporary measure

**Comment by shydesky** [comment]
Thanks for your PR! We will fix this problem with a hard-fork in the future. We do not only add the restrictions on the client-side. So we close this PR temporarily.

**Pull request closed without merging**